src/styles/blocks/card.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid $cardBorderColor;
+  background: $cardBgColor;
+
+  &__image-container {
+    display: flex;
+    justify-content: center;
+  }
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__heading {
+    margin: 0 0 4px;
+    color: $CardFontColorDark;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__code {
+    margin: 0 0 16px;
+    color: $CardFontColorLight;
+    font-size: 10px;
+    font-weight: 400;
```

Comment:
font-weight: 400; is a default browser value
```suggestion
```

Reply:
<img width="605" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/70220667/58c4ff8d-62d1-485b-abb4-a7d955ec56b8">